modules/__tests__/matchRoutes-test.js

```diff
@@ -332,6 +332,35 @@ describe('matchRoutes', function () {
     })
   })
 
+  describe('a nested route with a getChildRoutes callback', function () {
```

Comment:
Can you combine this with the previous `describe()` block? They're really covering the same thing.